add minus example to ceil function

alireza-mh opened the pull request:
Add minus integer number to ceil function.

jdalton commented:
Hi @alireza-mh!

Instead of an additional value example would you mind tweaking this to describe `ceil` as a similar to [`Math.ceil`](https://mdn.io/ceil). That way it provides more context for usage.

alireza-mh commented:
Hi @jdalton,
i updated the PR.
by adding that example i was intended to avoid some mathematical errors.
i added `rounded up` definition by borrowing it from MDN web doc.

jdalton commented:
Thank you @alireza-mh!
